src/components/listRow/index.tsx

```diff
@@ -0,0 +1,23 @@
+import {Skeleton, Text, View} from 'native-base';
+import React, {FC} from 'react';
+import styles from './styles';
+
+interface ListRowProps {
+  title: string;
+  value?: string | number;
```

Comment:
We dont need a default prop to `value` prop?

Reply:
I don't think so, actually, the logic intention is that if the component has an undefined value the loading state should be triggered. That component makes no sense with no value, but the value prop might be undefined while loading if it comes from an async operation